example-app/SantokuApp/src/components/picker/SelectPicker.tsx

```diff
@@ -40,7 +40,7 @@ export type SelectPickerProps<ItemT> = {
   /**
    * アイテムが選択された場合に呼び出される関数
    */
-  onSelectedItemChange?: (itemIndex: number, itemValue?: ItemT, itemKey?: React.Key) => void;
+  onSelectedItemChange?: (selectedItem: Item<ItemT> | undefined) => void;
```

Comment:
好み感はあるんだけど、こう書いといていいんじゃないかと。（他のやつも同じく揃えたほうがいいかなぁ、って）
```suggestion
  onSelectedItemChange?: (selectedItem?: Item<ItemT>) => void;
```

Reply:
対応しました。
[dd87dd5](https://github.com/ws-4020/mobile-app-crib-notes/pull/846/commits/dd87dd5d19c9b647d3909f4fafc1d6eb16084314)